src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <!-- <div class="card__container container"> -->
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="i-mac"
+          class="img"
+        >
+      </div>
+      <div class="card__description description">
+        <div class="item-card">
+          <div class="item-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="item-card__product-code">Product code: 195434</div>
+        </div>
+      </div>
+      <div class="card__stars">
+        <div class="element-card">
+          <div class="element-card__stars element-card__stars--not-active">
+            <div class="element-card__star"></div>
+            <div class="element-card__star"></div>
+            <div class="element-card__star"></div>
+            <div class="element-card__star"></div>
+            <div class="element-card__star"></div>
+          </div>
+          <div class="element-card__reviews">Reviews: 5</div>
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="price-card">
+          <div class="price-card__text">Price:</div>
+          <div class="price-card__price">$2,199</div>
+        </div>
+      </div>
+      <div class="card__button">
+        <div class="button-card">
+          <button
+            type="submit"
+            name="buy"
+            class="button-card__buy"
+          >
+            <a href="#buy"
+              data-qa="hover"
+              class="button-card__link"
+            >
```

Comment:
please check the attributes. each should start from new line
```suggestion
            <a
              href="#buy"
              data-qa="hover"
              class="button-card__link"
            >
```